packages/web-app-files/tests/unit/mixins/actions/restore.spec.js

```diff
@@ -79,8 +102,21 @@ function getWrapper({ invalidLocation = false, resolveClearTrashBin: resolveRest
       },
       $gettext: jest.fn(),
       $gettextInterpolate: jest.fn(),
+      createModal: jest.fn(),
+      $clientService: {
+        webdav: {
+          listFiles: jest.fn().mockImplementation(() => {
+            return []
+          })
+        }
+      },
```

Comment:
I understand if you don't want to port everything to the defaultComponentMocks, but you could mock the whole clientService like this:

https://github.com/owncloud/web/blob/3808ee02ee77549bb6133151f4fe15ce80fd2d95/tests/unit/mocks/defaultComponentMocks.ts#L20

Reply:
doesnt work 😬

Reply:
I doubt that it just `doesn't work` 👀  any specific reason why it `doesn't work`?